cmake/Platform/Mac/Install_mac.cmake

```diff
@@ -6,6 +6,31 @@
 #
 #
 
+#template for generating the project build_path setreg
```

Comment:
a more detailed comment would be welcomed here, something explaining that we generate binaries that call the ones in the bundles so dylibs outside of the bundle can be loaded, etc

Reply:
I'll add a comment to explain this and move the framework check in Install_common.cmake to the Mac file.